app/lib/common/themes/colors/color_scheme.dart

```diff
@@ -12,6 +12,11 @@ Color backgroundColor = const Color(0xff0d0f11);
 //On colors
 Color whiteColor = Colors.white;
 
+Color pinFeatureColor = const Color(0xff7c4a4a);
+Color eventFeatureColor = const Color(0xff206a9a);
+Color taskFeatureColor = const Color(0xff406c6e);
+Color boastFeatureColor = Colors.blueGrey;
```

Comment:
Stored this colors in common so that it can be used anywhere based on the requirements.